use_cases/transational_templates.md

```diff
@@ -66,6 +66,38 @@ print(response.body)
 print(response.headers)
 ```
 
+### With dynamic templates
```

Comment:
Might be missing dynamic content updates in the rest of the templates file

Reply:
You can check out some of the changes made [here](https://github.com/sendgrid/sendgrid-python/pull/597/files#diff-fa3865ac405555d68b7dd54aff4ea7ff) in #597

Reply:
Regarding updates for rest of file: this is left on purpose, as old template system is still in place, as well as code to support it. So, I guess we still need documentation for it.

Or I'm missing the point